target/linux/pistachio/base-files/usr/bin/get_mac.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
```

Comment:
It is strange to see such mac address generation (not only this file but others too)... Can you please give some explanation why you decided to do it this way.